third_party/library.sh

```diff
@@ -123,11 +123,36 @@ function generate() {
   generate_manifests "$path" "$@"
 }
 
+function run_yq() {
+  run_go_tool github.com/mikefarah/yq/v4 yq "$@"
+}
+
 function install_yaml() {
-  if grep -q "CustomResourceDefinition" "$1"; then
-    kubectl apply --selector="knative.dev/crd-install=true" -f "$1"
-    kubectl wait --for=condition=Established --all crd
```

Comment:
We will not wait for `Established` condition for CRD anymore. Is it alright?

Reply:
Yeah - i dropped it - I want to see if it flaked in the presubmit. N `kubectl` invocation is really slow so I figured it wouldn't come up